tools/AWSFeedStorage.py

```diff
@@ -17,22 +19,63 @@ def __init__(self, uri):
 
         self.logger = logging.getLogger(__name__)
 
+        u = urlparse(uri)
+        self.s3_file = u.path[1:]
         self.s3_bucket = settings['AWS_S3_BUCKET']
-        self.s3_file = settings['AWS_S3_FILE_NAME']
-        self.s3 = boto3.resource('s3')
+        self.s3 = boto3.client('s3')
         self.dynamodb = DynamoDBConnector()
 
+    def _generate_datetag(self):
+        curdate = datetime.now()
+        return "{}".format(curdate.strftime('%Y%m%d'))
```

Comment:
The `str.format` is totally redundant: this would be simpler as:

```python
def _generate_datetag(self):
    return date time.now().strftime('%Y%m%d')
```

And you could make this a staticmethod since it doesn’t use `self`.